src/plugins/fileViewer/fileViewer.css

```diff
@@ -0,0 +1,29 @@
+.file-viewer.container {
```

Comment:
classnames should be prefixed with `vc-` to differentiate them from discord classes and avoid conflicts

```suggestion
.vc-fileViewer.container {
```